chart/develop/templates/deployment_hpa.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
```

Comment:
In addition to an HPA, can you also add a PodDisruptionBudget? (both deployment and the worker)

Reply:
yes you're right althoguht i have no clue how to do that

Reply:
Ok Fixed.